includes/class-doubleclick.php

```diff
@@ -153,20 +153,22 @@ public function enqueue_scripts() {
 
 		$data = array(
 			'network_code' => $this->network_code,
-			'mappings' => $mappings,
+			'mappings' => apply_filters( 'dfw_mappings', $mappings ),
 			'targeting' => $this->targeting(),
 		);
 
 		wp_localize_script( 'jquery.dfw.js', 'dfw', $data );
 		wp_enqueue_script( 'jquery.dfw.js' );
 
-		wp_enqueue_style(
-			'dfp',
-			plugins_url( 'assets/css/dfp.css', dirname( __FILE__ ) ),
-			array(),
-			DoubleClick_For_WordPress::VERSION,
-			'all'
-		);
+		if ( ! empty( get_option( 'dfw_css' ) ) ) {
```

Comment:
yes, in https://github.com/INN/doubleclick-for-wp/blob/plugin-boilerplate/includes/class-options.php#L199

Hold off on merging this for a bit though, I need to add a thing to fire dfw_load(), probably in jquery.dfw.js on document ready but I want to make sure the timing is correct and a callback exists because we have some stuff we need to run before that loads so I just need to do a little more testing.